src/main/webapp/app/assessment/assessment-instructions/assessment-instructions/assessment-instructions.component.ts

```diff
@@ -55,4 +59,7 @@ export class AssessmentInstructionsComponent {
             this.sampleSolution = this.markdownService.safeHtmlForMarkdown(sampleSolutionMarkdown);
         }
     }
+    drag(ev: any, instruction: GradingInstruction) {
```

Comment:
please add a JSDoc comment here, that this method connects the GradingInstruction with the Feedback